2021/27xxx/CVE-2021-27759.json

```diff
@@ -1,18 +1,87 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "PSIRT@hcl.com",
+        "DATE_PUBLIC": "2022-04-28T00:00:00.000Z",
         "ID": "CVE-2021-27759",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC"
     },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "HCL BigFix Inventory",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "9.x"
+                                        },
+                                        {
+                                            "version_value": "10.x"
+                                        }
+                                    ]
+                                }
+                            }
+                        ]
+                    },
+                    "vendor_name": "HCL Software"
+                }
+            ]
+        }
+    },
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "This vulnerability arises because the application allows the user to perform some sensitive action without verifying that the request was sent intentionally. An attacker can cause a victim's browser to emit an HTTP request to an arbitrary URL in the application.\n"
+            }
+        ]
+    },
+    "generator": {
+        "engine": "Vulnogram 0.0.9"
+    },
+    "impact": {
+        "cvss": {
+            "attackComplexity": "HIGH",
+            "attackVector": "ADJACENT_NETWORK",
+            "availabilityImpact": "NONE",
+            "baseScore": 1.6,
+            "baseSeverity": "LOW",
+            "confidentialityImpact": “LOW”,
```

Comment:
The qoutes for the values of confidentialityImpact, privilegesRequired, and userInteraction are not accepted. This usually happens from a copy/paste. Just remove quotes and reapply quotes in your editor.